ui/components/app/assets/asset-list/asset-list-control-bar/index.scss

```diff
@@ -0,0 +1,12 @@
+.asset-list-control-bar {
+  display: flex;
+  justify-content: space-between;
```

Comment:
Can you use style utility props instead of styles here?

Reply:
https://github.com/MetaMask/metamask-extension/pull/27084/commits/4baa9ad7f67c1daf8b359232dd7dbbe3902d84a7